projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.html

```diff
@@ -1,238 +1,134 @@
-                        </div>
-                      </div>
-                      <div
-                        class="title text-left d-flex justify-content-around"
-                        [ngClass]="
-                          policy.billingData[0]?.paymentStatus == 'Overdue'
-                            ? 'warn'
-                            : 'notice'
-                        "
-                      >
-                        {{ policy.billingData[0]?.paymentAmount }}
-                      </div>
-                    </div>
+            <div class="container-fluid">
+              <ng-container
+                *ngFor="let dueDate of policy.billingData | slice: 1"
+              >
+                <div class="row">
+                  <div class="col"></div>
+                  <div class="col"></div>
+                  <div class="col color-dot text-left mw-md-100">
+                    {{ dueDate.dueDate | cxDate: 'dd MMM yyyy' }}
                   </div>
-                </div>
-                <ng-container *ngIf="policy.billingData.length > 1">
-                  <div
-                    class="accordion container"
-                    [ngClass]="policy.opened ? 'opened' : ''"
-                  >
-                    <ul class="list-data">
-                      <ng-container
-                        *ngFor="let dueDate of policy.billingData | slice: 1"
-                      >
-                        <li class="row list-item border-bottom border-color-3">
-                          <div
-                            class="col-6 text-left d-flex justify-content-around"
-                          >
-                            {{ dueDate.dueDate | cxDate: 'dd MMM yyyy' }}
-                          </div>
-                          <div
-                            class="col-6 text-left d-flex justify-content-around"
-                          >
-                            {{ dueDate.paymentAmount }}
-                          </div>
-                        </li>
-                      </ng-container>
-                    </ul>
+                  <div class="col color-dot text-left">
+                    {{ dueDate.paymentAmount }}
                   </div>
-                </ng-container>
-                <div class="cards__data">
-                  <span class="title">
-                    {{ 'fscommon.paymentMethod' | cxTranslate }}
-                  </span>
-                  <ng-container *ngIf="policy.paymentMethod; else NA">
-                    {{ policy.paymentMethod.cardType?.code | uppercase }}
-                    {{ policy.paymentMethod?.cardNumber }}
-                  </ng-container>
+                  <div class="col"></div>
                 </div>
-              </div>
+              </ng-container>
             </div>
           </div>
-        </ng-container>
+          <div class="col border border-top-0 d-lg-none text-center">
+            <h5 class="d-lg-none m-0">
```

Comment:
`d-lg-none` is not needed here I would say, since the wrapping div has the same class